README.md

```diff
@@ -1,222 +1,515 @@
 # jsgreat
-[![npm version](https://badge.fury.io/js/jsgreat.svg)](http://badge.fury.io/js/jsgreat)
 
-Our JavaScript code will (mostly) follow the [Google JavaScript Style Guide](http://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml). A few slight variations and additions are laid out below. There will also be exceptions to these rules in some existing code, but where possible, existing code should be updated as it is edited to conform to the style guide.
+[![npm Version](https://img.shields.io/npm/v/jsgreat.svg?style=flat-square)](https://www.npmjs.com/package/jsgreat)
+[![MIT License](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](https://github.com/yola/jsgreat/master/LICENSE)
+
+## Table of Contents
+
+* [Introduction](#introduction)
+  - [Changes](#changes)
+  - [Linting](#linting)
+  - [Installation](#installation)
+* [Rules](#rules)
+  - [Naming](#naming)
+  - [Variables](#variables)
+    * [Variable statements](#variable-statements)
+    * [Unused Variables](#unused-variables)
+    * [Undeclared Variables](#undeclared-variables)
+    * [Using Variables Before Declaration](#using-variables-before-declaration)
+    * [Aligning Assignments](#aligning-assignments)
+  - [Constants](#constants)
+  - [Functions](#functions)
+    * [Anonymous Functions](#anonymous-functions)
+  - [Modules](#modules)
+    * [Exporting](#exporting)
+    * [Importing](#importing)
+  - [Promises](#promises)
+  - [JSDoc](#jsdoc)
+  - [Visibility](#visibility)
+  - [Indentation](#indentation)
+  - [Manipulating `this` context](#manipulating-this-context)
+  - [Curly Braces](#curly-braces)
+  - [Strict Mode](#strict-mode)
+  - [Coerced Types and Equality](#coerced-types-and-equality)
+  - [Trailing Whitespace](#trailing-whitespace)
+  - [80 Column Limit](#80-column-limit)
+* [License](#license)
+
+## Introduction
+
+This guide outlines the style for Yola's JavaScript code.
+When editing existing code, and where possible, said code should be updated to conform to the style guide.
+
+### Changes
+
+Proposed style changes should be discussed in a written format that can referenced.
+The style guide must be updated to reflect all changes.
```

Comment:
I think you're trying to say that any changes made to the linter config must be described in the style guide, too.